pkg/minishift/provisioner/minishift_provisioner.go

```diff
@@ -88,6 +88,41 @@ func (provisioner *MinishiftProvisioner) Package(name string, action pkgaction.P
 	return errors.New("Custom provisioner does not support package")
 }
 
+func (provisioner *MinishiftProvisioner) remoteHostPrepration() error {
```

Comment:
s/remoteHostPrepration/PreparingRemoteHost/

Reply:
Preparng would also be wrong. `prepareRemoteHost` because `-ing` would suggest that the preparation step is only setup to be finished AFTER running an additional step after this function.